plugin/pkg/admission/nodeenv/admission.go

```diff
@@ -0,0 +1,117 @@
+package nodeenv
```

Comment:
The name for this plugin should be podplacementrestriction or namespacepodplacement or something.